_component_design/chips.md

```diff
@@ -17,27 +17,99 @@ helix-ui-javascript: false
 
 <section class="static-section" markdown="1">
 
-## Introduction
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
```

Comment:
These column calls all need to be updated before merging as the NPM library switch has shipped. So, change them throughout as follows:
```
{%.column <left/right>:"hxCol hxSpan-##-xs hxSpan-##-sm hxSpan-##-md hxSpan-##-lg" %}
```